### PR TITLE
Added Prime Number in C++

### PR DESCRIPTION
**Pull request by Akshay2903**
Congrats on taking the first step to contributing to the Sample Programs repository maintained by [The Renegade Coder][1]! For simplicity, please make sure that your pull request includes one and only one contribution.

## Complete the Applicable Sections Below

Find which section best describes your pull request and make sure you fill it out. To start, let us know which issue you've fixed.

- [x] I fixed #your-issue-number-here

### Code Snippets

- [x] I named the pull request using `Added/Updated <Sample Program> in <Language>` format
- [x] I created/updated the language README
  - [x] I added the sample program name to the README
  - [x] I added fun facts (i.e. debut, developer, typing, etc.)
  - [x] I added reference link(s) to the README
  - [x] I added solution citations when necessary (see [plagiarism][2])

### Documentation

- [ ] I named the pull request using `Added/Updated <Sample Program> in <Language> Article` format
- [ ] I followed the applicable article template
  - [ ] I added/updated a [language article][4] (i.e. The Python Programming Language)
  - [ ] I added/updated a [code snippet article][3] (i.e. Hello World in Perl)
  - [ ] I added/updated a [project article][5] (i.e. Fizz Buzz in Every Language)

### Testing

- [x] I named the pull request using `Added/Updated <Language>/<Project> Testing` format
- [x] I followed the testinfo template, if applicable

## Notes

Feel free to describe what you added or updated.

[1]: https://therenegadecoder.com/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#plagiarism
[3]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/CODE_ARTICLE_TEMPLATE.md
[4]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/LANGUAGE_ARTICLE_TEMPLATE.md
[5]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/PROJECT_ARTICLE_TEMPLATE.md

### CODE REVIEW

**Comment by Akshay2903** [comment]
hey TheRenegadeCoder/sample-programs i add it to the README

File: archive/c/c-plus-plus/README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Welcome to Sample Programs in C++!
 - [Reverse String in C++](https://github.com/TheRenegadeCoder/sample-programs/issues/419)
 - [Fibonacci Sequence in C++](https://github.com/TheRenegadeCoder/sample-programs/issues/496)
 - [Bubble Sort in C++](https://github.com/TheRenegadeCoder/sample-programs/issues/1135)
+- [prime_number in C++](https://github.com/TheRenegadeCoder/sample-programs/issues/1227)
```
Comment (jrg94) [inline archive/c/c-plus-plus/README.md]:
```suggestion
- [Prime Number in C++](https://github.com/TheRenegadeCoder/sample-programs/issues/1227)
```

File: archive/c/c-plus-plus/even_odd.cpp [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#include <iostream>
```
Comment (jrg94) [inline archive/c/c-plus-plus/even_odd.cpp]: Remove this file.

File: archive/c/c-plus-plus/README.md (hunk 1 above)
Comment (Akshay2903) [reply archive/c/c-plus-plus/README.md]: I changed the code according to the testing

**Comment by jrg94** [comment]
Looks like this got added in another pull request. Sorry about that!

**Pull request closed without merging**